gravity.sh

```diff
@@ -254,10 +254,10 @@ function gravity_hostFormat() {
 		echo "::: Error: Unable to determine fully qualified domain name of host"
 	fi
 	# If there is a value in the $piholeIPv6, then IPv6 will be used, so the awk command modified to create a line for both protocols
-	if [[ -n ${piholeIPv6} ]];then
```

Comment:
I'm working mostly on `basic-install.sh` haven't gotten to much into `gravity.sh`